repo/packages/I/instana-agent/2/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "5.0",
+  "minDcosReleaseVersion": "1.8",
+  "name": "instana-agent",
+  "version": "1.0.5",
+  "maintainer": "support@instana.com",
+  "website": "https://www.instana.com",
+  "description": "Instana is the next generation Application Performance Management solution that automatically monitors scheduled environments/dynamic modern applications.\nInstana comprehensively manages the quality of service of your microservice applications with zero to absolute minimal configuration effort.",
+  "preInstallNotes": "This DC/OS Service is currently in preview. This package will deploy the Instana agent on your Mesos nodes.",
```

Comment:
remove this line from `preInstallNotes` - `This DC/OS Service is currently in preview. `